src/AlgebraicGeometry/Schemes/Sheaves/IdealSheaves.jl

```diff
@@ -345,7 +345,7 @@ end
 end
   
 @attr Bool function has_dimension_leq_zero(I::Ideal)
-  is_one(I) && return true
+  #is_one(I) && return true
```

Comment:
I am sure you have good reasons for this change and I appreciate your efforts to improve things, thank you!

However, whenever I see code being commented out and still being committed, I wonder: why is that?

As a rule of thumb, it almost always seems to be down to one of the following two:
1. it is wrong or for some reason not needed anymore.
2. it is desirable on the long run (e.g. an optimization) but for some reason has to be disabled for the time being (e.g. some other code needs a fix)

In the first case, we are better off just deleting the code.

In the second case, I really think a comment should be added explaining what's going on. Because right now, *you* certainly have the answer to this in your mind. But anyone looking at this code in a year will not, and they'll be thankful for any included hints...

Thus it would be super nice if you could do one or the other, and likewise in line 357 when the same happens :-)